sleap/gui/color.py

```diff
@@ -295,5 +331,11 @@ def get_item_color(
                         break
 
             return self.get_color_by_idx(edge_idx)
+        
+        if self.distinctly_color == "InstanceGroups" and parent_session and parent_frame_idx:
+            if isinstance(item, Instance):
+                frame_group = parent_session.frame_groups[parent_frame_idx]
+                instance_group = frame_group.get_instance_group(instance=parent_instance)
+                return self.get_instance_group_color(instance_group)
```

Comment:
So right now, we color the plotted `Instance`s either by the entire `Instance` (nodes and edges all one color), individual `Node`s (all nodes within an `Instance` are different colors), or by `Edge`s (all edges within an `Instance` are different colors). For our color by `InstanceGroup` option, we still want the logic to be within the 
```
if self.distinctly_color == "instances"
```
conditional, but now instead of just having logic to handle coloring by `Track`s
https://github.com/talmolab/sleap/blob/0134ab443663bf1b1f0287d632dce9fdef80c14b/sleap/gui/color.py#L260-L271
we want to expand this block to also include logic for coloring by `InstanceGroup`s. Get rid of the block you have here and try to expand the existing `if self.distinctly_color == "instances"` conditional.